packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -38,6 +38,7 @@ export interface RawRumResourceEvent {
   _dd?: {
     trace_id?: string
     span_id?: string // not available for initial document tracing
+    rule_psr?: number
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏we should also add that to the rum-event-format